tests/ops/integration_tests/test_attentive_task.py

```diff
@@ -25,8 +24,25 @@ async def test_non_strict_erasure_request(
             erasure_policy=erasure_policy_string_rewrite,
             identities={
                 "email": attentive_erasure_identity_email,
+            },
+        )
+        assert erasure_results == {"attentive_instance:user": 1}
+
+    async def test_non_strict_erasure_request_with_phone_number(
```

Comment:
I added two separate tests to verify that the delete will work with either email or phone number